core/transfer/src/main/java/org/eclipse/dataspaceconnector/transfer/core/transfer/DelegatingTransferProcessManager.java

```diff
@@ -30,6 +33,12 @@ public TransferInitiateResult initiateProviderRequest(DataRequest dataRequest) {
         return dataRequest.isSync() ? syncManager.initiateProviderRequest(dataRequest) : asyncManager.initiateProviderRequest(dataRequest);
     }
 
+    @Override
+    public CompletableFuture<Void> enqueueCommand(Command command) {
+        return asyncManager.enqueueCommand(command);
+        //todo: should the sync manager also have a command queue?
```

Comment:
in my opinion the `Sync` process manager should be merged with the async, adding an if condition on the `initiate` methods and setting different behaviours for different `DataRequest` type, because the empty-implementation methods on the sync one are telling us that maybe that was not the best design. Should be addressed on another issue, but I think that's why some questions like this comes to the surface

Reply:
Opened an issue about: #509